packages/sysvars/src/__tests__/epoch-rewards-test.ts

```diff
@@ -0,0 +1,19 @@
+import { getSysvarEpochRewardsCodec } from '../epoch-rewards';
+
+describe('epoch rewards', () => {
+    it('decode', () => {
+        // prettier-ignore
+        const epochRewardsState = new Uint8Array([
+            0, 45, 49, 1, 0, 0, 0, 0,    // distributionCompleteBlockHeight
+            134, 74, 2, 0, 0, 0, 0, 0,   // distributedRewards
+            0, 132, 215, 23, 0, 0, 0, 0, // totalRewards
+        ]);
+        expect(getSysvarEpochRewardsCodec().decode(epochRewardsState)).toMatchObject({
+            distributedRewards: 150_150n,
+            distributionCompleteBlockHeight: 20_000_000n,
+            totalRewards: 400_000_000n,
+        });
+    });
+    // TODO: This account does not seem to exist on-chain yet.
+    it.todo('fetch');
```

Comment:
Will it ever be possible to fetch that accounts in the tests? If not, I'd say let's write a comment but ditch the `todo` test.

Reply:
It would be if we wanted to tell our test validator to start warped to epoch 1, which might not be a bad idea for a lot of reasons. Maybe I can see if doing that breaks anything?